carma_wm_ctrl/test/WMBroadcasterTest.cpp

```diff
@@ -58,13 +58,13 @@ namespace carma_wm_ctrl
 
 {
 
-TEST(WMBroadcaster, Constructor)
+TEST(WMBroadcaster, DISABLED_Constructor)
```

Comment:
Most of the unit tests in this file have been disabled. Let's undo that.

Reply:
enabled